src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working.");
```

Comment:
The stop message could also be more informative. It should clearly indicate that it's an 'Excavator' that has stopped working, such as 'The excavator has ceased operations.'